articles/150_ros_command_line_arguments.md

```diff
@@ -167,16 +167,25 @@ another_node:
      string_param: bar
 ```
 
-#### Logging level
+#### Logging level assignments
 
-Minimum logging level can be externally set using the `--log-level` option.
+Minimum logging level can be externally set either globally or per logger using the `--log-level` option.
 
-As an example, to set logging level to `DEBUG` for `some_ros_executable`, one may execute:
+As an example, to set a global logging level to `DEBUG` for `some_ros_executable`, one may execute:
 
 ```sh
 ros2 run some_package some_ros_executable --ros-args --log-level DEBUG
 ```
 
+Some specific loggers can be set using the `--log-level` option as well:
+
+```sh
+ros2 run some_package some_ros_executable --ros-args --log-level talker1:=DEBUG --log-level talker2:=WARN
```

Comment:
It might be nice to have an example that's not just a node, like maybe `rclcpp` or `talker1.rclcpp` or something more interesting like that.